state-synchronizer/src/tests/on_chain_config_tests.rs

```diff
@@ -72,18 +75,10 @@ fn test_on_chain_config_pub_sub() {
     // Case 2: publish if subscribed config changed //
     //////////////////////////////////////////////////
     let genesis_account = libra_root_address();
-    let network_config = config.validator_network.as_ref().unwrap();
-    let validator_account = network_config.peer_id();
-    let operator_key = config
-        .test
-        .as_ref()
-        .unwrap()
-        .operator_key
-        .as_ref()
-        .unwrap()
-        .private_key();
+    let validator_account = validators[0].owner_address;
+    let operator_key = validators[0].key.clone();
```

Comment:
not very clear whether it's owner key or operator key or consensus key

Reply:
all keys are the same <3